Homework/src/by/tms/Lesson_7/appliances/Blender.java

```diff
@@ -0,0 +1,25 @@
+package by.tms.Lesson_7.appliances;
+
+public final class Blender extends KitchenAppliances {
+
+    private String brand;
+
+    public Blender(String brand) {
+        super("Blender");
+        this.brand = brand;
+    }
+
+    public String getBrand() {
+        return brand;
+    }
+
+    public void setBrand(String brand) {
+        this.brand = brand;
+    }
+
+    @Override
+    public String toString() {
```

Comment:
It's better to call super.toString() and include response in current method.
Because if number/type/count of fields in superclass will be changed - you'll need to overwrite all toString() methods in inherited classes
apply this solution to all inherited classes